easybuild/easyconfigs/g/GTDB-Tk/GTDB-Tk-1.7.0-foss-2020b.eb

```diff
@@ -0,0 +1,43 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'GTDB-Tk'
+version = '1.7.0'
+
+homepage = 'https://github.com/Ecogenomics/GTDBTk'
+description = "A toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes."
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://pypi.python.org/packages/source/g/gtdbtk']
+sources = ['gtdbtk-%(version)s.tar.gz']
+checksums = ['36b65074845e7edef4d7aeead286c770fb9a03ddbff63946533fa2e1934e8f26']
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('DendroPy', '4.5.2'),
+    ('matplotlib', '3.3.3'),
+    ('prodigal', '2.6.3'),
+    ('HMMER', '3.3.2'),
+    ('pplacer', '1.1.alpha19', '', True),
+    ('FastANI', '1.33'),
+    ('FastTree', '2.1.11'),
+    ('tqdm', '4.56.2'),
+]
+
+download_dep_fail = True
+use_pip = True
+
+options = {'modulename': 'gtdbtk'}
+
+sanity_check_paths = {
+    'files': ['bin/gtdbtk'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+

```

Comment:
@stevecdavis is there a simple `sanity_check_command` we could add? E.g. `--help` or `--version`?

Reply:
I don't think so.... When I run the -h command it's telling me that:
`The 'GTDBTK_DATA_PATH' environment variable is not defined`
But according to their site,
`GTDB-Tk requires ~27G of external data that needs to be downloaded and unarchived:`

I believe the end users set that path when they run their jobs.

I do get the error message as a response for the software, would that work as a sanity check though? (also there weren't checks in other versions I looked at, not sure if that matters)

Reply:
@stevecdavis don't worry about it, if it can't be added quickly and easily.

It's something we're trying to add as we go along, as a more robust check that the build has worked. That's why you may not have seen it in earlier versions.